db/1.8/modules/ROOT/pages/database-documentation.adoc

```diff
@@ -738,6 +738,16 @@ Invokes a stored procedure on the database.  When the stored procedure returns o
 === For Configurations
 * <<config>>
 
+=== Working with Pooling Profiles
+
+The Stored procedure operation closes the connections it receives once the flow containing the operation completes. This behavior occurs because the resultset the operation generates can have a stream or be part of an ongoing transaction. Thereby, Mule runtime engine never knows when the flow is going to consume the resultset in its entirety nor when the connection is going to be needed.
```

Comment:
```suggestion
The *Stored procedure* operation closes the connections it receives when the flow that contains the operation completes. This behavior occurs because the resultset the operation generates can have a stream or be part of an ongoing transaction. Thereby, Mule runtime engine never knows when the flow is going to consume the resultset in its entirety, nor when the connection will be needed.
```